packages/plans/components/PlanColumnWithPremiumSolo/index.jsx

```diff
@@ -107,11 +121,12 @@ const PlanColumnWithPremiumSolo = ({
   billingText,
   onPremiumPlanClick,
   selectedPremiumPlan,
+  isAwesomePromoUser,
 }) => {
   const isSelected = () => selectedPremiumPlan === 1;
   const isPremium = plan === 'premium_business';
   return (
-    <ColumnStyle>
+    <ColumnStyle isAwesomeUser={isAwesomePromoUser}>
```

Comment:
I'd vote to keep the naming consistent here. Since not _all_ Awesome users will see this.

```suggestion
    <ColumnStyle isAwesomePromoUser={isAwesomePromoUser}>
```